custom_components/thermal_comfort/sensor.py

```diff
@@ -520,7 +530,7 @@ def __init__(
 
         if self._should_poll:
             if scan_interval is None:
-                scan_interval = timedelta(seconds=30)
+                scan_interval = timedelta(seconds=SCAN_INTERVAL_DEFAULT)
```

Comment:
Thanks for adding constants, totally makes sense. I just rushed to get a working fix out. Sorry